Separate RTP and RTCP packets from the server

RogerHardiman opened the pull request:
This change splits out the RTP and the RTCP packets that are received from the server and is the first step to passing only RTP packets into code that will reconstruct the H264 video stream.

The PR also fixes an issue if the header has a non zero CSRC count.

mbullington commented on lib/index.js:
```diff
@@ -26,16 +26,19 @@ class RtspClient extends EventEmitter {
     this.originalPacketLength = -1;
     this.packetLength = -1;
     this.packets = [];
+    this.packet_channel_id = -1;
```
For consistency with codebase, I wouldn't mind having this be named packetChannelId, or just packetChannel.

RogerHardiman replied:
Good idea

mbullington commented on lib/index.js:
```diff
@@ -46,19 +49,26 @@ class RtspClient extends EventEmitter {
         this.packets = [];
 
-        this.emit("data", packet.payload, packet);
+        this.emit("data", this.packet_channel_id, packet.payload, packet);
```
Is this really the cleanest thing, API wise, that we could do in this situation? Since from what I can see from RTSP specification this will only ever be zero or one, could a separate event for RTCP, like maybe "controlData", be fired instead? I think it would remove some possible confusion.

RogerHardiman replied:
Having an event for RTP and another for RTCP is a good idea.
We still need the channel. If you were requesting video and audio then they would be in different channels.

RogerHardiman commented:
I have updated the code with your suggestions.
I have also added parsing the Marker Bit to the RTP payload and the Payload Type to the RTCP payload and display them in the wows example.

Marker bit is used when a block of video (e.g. a H264 I-frame) is split over several RTP packets.
RTCP payload type shows that Wowza is sending message 200 which is the Sender Report.

mbullington commented:
Great, merging into main branch, it should be on npm in a release soon after. Thank you!
